schemas/telemetry/_common-schema.json

```diff
@@ -92,6 +92,14 @@
           "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
         }
       }
+    },
+    "experimental_features":  {
```

Comment:
Indexing this new value requires a minor update of the backend.
Speaking of indexing, do you want the list to be stored in the log's body as is?